File: ForRadiant/FolderLock/Password.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using static System.Windows.Forms.VisualStyles.VisualStyleElement;

namespace FolderLock
{
    public partial class Password : Form
    {
        public string password = "";
        public Password()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            bool passwordEntered = false;
            this.password = textBox1.Text;
            if (this.password == "")
            {
                MessageBox.Show("Password cannot be blank");
            }
            else passwordEntered = true;

            if (passwordEntered == true)
            {
                this.Close();
            }
        }

        private void textBox1_KeyDown(object sender, KeyEventArgs e)
        {
            if (e.KeyCode == Keys.Enter)
            {
                button1_Click(this, new EventArgs());
            }

            if (e.KeyCode == Keys.Escape)
            {
                this.Close();
            }


        }
    }
}

File: ForRadiant/CopyMaster/Password.vb
Imports System
Imports System.Runtime.CompilerServices
Imports System.Windows.Forms
Imports CopyMasterAppData.CopyMasterAppData
Imports Microsoft.VisualBasic

Public Class Password
    Public Property password As String = ""

    Private Sub Button1_Click(sender As Object, e As System.EventArgs) Handles Button1.Click
        Dim passwordEntered As Boolean
        password = TextBox1.Text
        If password = "" Then
            MessageBox.Show("Password cannot be blank")
        Else
            passwordEntered = True
        End If
        If passwordEntered Then
            Me.Close()
        End If

    End Sub

    Private Sub TextBox1_KeyDown(sender As Object, e As KeyEventArgs) Handles TextBox1.KeyDown
        If (e.KeyCode = Keys.Enter) Then

            Button1_Click(Me, New EventArgs())
        End If

        If (e.KeyCode = Keys.Escape) Then
            Me.Close()
        End If

    End Sub

End Class